.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ jobs:
             gem install bundler
             bundle install --path vendor/bundle
             sudo apt install lsof
-            bundle exec spinach --tags simple-get
+            bundle exec spinach --tags simple-get, head-to-simple-get
```

Comment:
I don't think the tag `head-to-simple-get` exists, might just be `simple-head`. https://github.com/8thlight/http_server_spec/blob/master/features/01_getting_started/simple_head.feature

Reply:
Make sense, no worries!